mods/cnc/rules/campaign-maprules.yaml

```diff
@@ -40,6 +42,21 @@ Player:
 		Name: Campaign Player AI
 		Type: campaign
 
+^ExistsInWorld:
+	ScriptTriggers:
+
+^CivBuildingHusk:
```

Comment:
Aren't these already handled by `^ExistsInWorld`?

Reply:
No, cnc `^CivBuildingHusk` doesn't inherit that since most of the stuff on ExistsInWorld is redundant for them.